### PR TITLE
Adds missing changelog entries

### PR DESCRIPTION
**Pull request by vinay-gopalan**
Adds missing CL entries for v0.19.2, v0.19.1 and v0.17.3 that were released on Jul 1.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,20 @@
 FEATURES:
 * Adds ability to limit the lifetime of service principal secrets in Azure through `explicit_max_ttl` on roles ([GH-199](https://github.com/hashicorp/vault-plugin-secrets-azure/pull/199))
 
+## v0.19.2
+IMPROVEMENTS:
+* Updated dependencies [[GH-215]](https://github.com/hashicorp/vault-plugin-secrets-azure/pull/215)
+  * `github.com/Azure/azure-sdk-for-go/sdk/azidentity` v1.5.2 ->  v1.6.0
+  * `github.com/hashicorp/go-retryablehttp` v0.7.1 -> v0.7.7
```
Comment (benashz) [inline CHANGELOG.md:10]: Were these updated under the PR above?

**Review by fairclothjm — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -41,6 +55,22 @@ IMPROVEMENTS:
...
 * `golang.org/x/net` 0.21.0 -> 0.23.0 [[GH-195]](https://github.com/hashicorp/vault-plugin-secrets-azure/pull/195)
 
+## v0.17.3
+
+BUG FIXES:
+* Fix segmentation fault when unassigning role assignments [[GH-213]](https://github.com/hashicorp/vault-plugin-secrets-azure/pull/213)
+
+IMPROVEMENTS:
+* Update dependencies [[GH-214]](https://github.com/hashicorp/vault-plugin-secrets-azure/pull/214)
+  * `github.com/Azure/azure-sdk-for-go/sdk/azcore` v1.9.1 -> v1.11.1
```
Comment (benashz) [inline CHANGELOG.md:65]: Similar question as above.

**Review by benashz — COMMENTED** [review]
Overall looks good. Just had some questions around PR attribution.

File: CHANGELOG.md (hunk 1 above)
Comment (vinay-gopalan) [reply CHANGELOG.md:10]: Yup! https://github.com/hashicorp/vault-plugin-secrets-azure/pull/215/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R52

File: CHANGELOG.md (hunk 2 above)
Comment (vinay-gopalan) [reply CHANGELOG.md:65]: Yes here as well: https://github.com/hashicorp/vault-plugin-secrets-azure/pull/214/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R52

**Pull request merged**